zenoh/src/lib.rs

```diff
@@ -193,7 +193,7 @@ pub mod session {
         builders::publisher::{SessionDeleteBuilder, SessionPutBuilder},
         info::{PeersZenohIdBuilder, RoutersZenohIdBuilder, SessionInfo, ZenohIdBuilder},
         query::SessionGetBuilder,
-        session::{open, OpenBuilder, Session, Undeclarable},
+        session::{open, OpenBuilder, Session, SessionClosed, Undeclarable},
```

Comment:
This should be named `SessionCloseError` I suppose, by analogy with `ReplyError`, `ZAllocError`

Reply:
Done